packages/pwa-buildpack/lib/Utilities/createFileHash.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const crypto = require('crypto');
+
+function createFileHash(filePath) {
+    const result = new Promise(resolve => {
+        try {
+            if (fs.existsSync(filePath)) {
+                var fd = fs.createReadStream(filePath);
+                var hash = crypto.createHash('md5');
+                hash.setEncoding('hex');
+
+                fd.on('end', function() {
+                    hash.end();
+                    resolve(hash.read());
+                });
+
+                fd.on('error', function() {
+                    hash.end();
+                    console.error(`Unable to read file ${filePath}`);
+                    resolve('');
+                });
+
+                fd.pipe(hash);
+            } else {
+                throw new Error(`Error reading file ${filePath}`);
+            }
+        } catch (err) {
+            console.error(`Unable to create hash of ${filePath}`, err);
+            resolve('');
```

Comment:
We shouldn't because this is not mandatory and should not stop the build process by throwing an error. If we are unable to create a hash, we have to create a random string and insert because the file name structure of `filename.[hash].fileextention` should be maintained for consistency.

Reply:
Yeah, I have changed the implementation recently to reflect the changes. Instead of 'something_else' it will be a random string for each call. This fallback will be just like the previous implementation because previously as well, we used to get a new file on every compile.